ios/Handlers/LoginHandler.swift

```diff
@@ -1,19 +1,20 @@
 protocol BridgeToLoginHandlerProtocol: AnyObject {
     func authenticateUser(_ profile: ONGUserProfile, authenticator: ONGAuthenticator?, completion: @escaping (ONGUserProfile?, NSError?) -> Void)
+    func setAuthPinChallenge(_ challenge: ONGPinChallenge?)
+    func handlePinAction(_ pin: String, action: String)
```

Comment:
why action is a string and not PinAction?

Reply:
they need to be mapped from a string -> PinAction somewhere, that is in this function, which is the first function call done by the wrapper.

Reply:
it does not matter, it still can (and should) be PinAction here, and in the place you are passing it to the wrapper just call pinAction.rawValue